### PR TITLE
feat: navbar style and stots

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Because we introduced the menu and removed tabs, the page name isn't displayed on views anymore. Therefore this PR remove the static "NETWORK NERVOUS SYSTEM" and replaces it with a dynamic title.

In addition, navbar used to be different on main screens as on details page. This PR aligns both style - i.e. one header / navbar style and behavior for the all app.

Finally, the "logout" option was not available on the detail page, so this PR makes the option available too.

# Changes

- make the title in `Header` dynamic
- refactor `HeadlessLayout` to use common `Header`
- refactor `Toolbar` to use it in `footer` (as existing) and newly in `header` too

# Screenshots

Capture d’écran 2022-06-20 à 12 58 45
Capture d’écran 2022-06-20 à 12 58 48
Capture d’écran 2022-06-20 à 12 58 53
Capture d’écran 2022-06-20 à 12 58 55
Capture d’écran 2022-06-20 à 12 59 00
Capture d’écran 2022-06-20 à 12 59 02
gif

### CODE REVIEW

File: frontend/svelte/src/lib/themes/variables.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @use "./mixins/media";
 :root {
-  --header-height: 76px;
+  --header-height: 60px;
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/themes/variables.scss:3]: Probably `--padding-6x` will be more flexible with themes.

File: frontend/svelte/src/tests/lib/components/common/Header.spec.ts [hunk 2]
```diff
@@ -7,25 +7,11 @@ import Header from "../../../../lib/components/common/Header.svelte";
 import en from "../../../mocks/i18n.mock";
 
 describe("Header", () => {
-  it("should render a title", () => {
-    const { getByText } = render(Header);
-
-    expect(getByText("NETWORK NERVOUS SYSTEM")).toBeInTheDocument();
-  });
-
   it("should render a logout button", () => {
```
Comment (mstrasinskis) [inline frontend/svelte/src/tests/lib/components/common/Header.spec.ts:10]: nth: should render a slot content

**Review by mstrasinskis — APPROVED** [review]
Thanks, great UX improvement 👍
Just a couple of minor suggestions.

File: frontend/svelte/src/lib/themes/variables.scss (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/lib/themes/variables.scss:3]: As discussed today, `--padding-6x` was actually equals to `--padding-4x`

File: frontend/svelte/src/tests/lib/components/common/Header.spec.ts (hunk 2 above)
Comment (peterpeterparker) [reply frontend/svelte/src/tests/lib/components/common/Header.spec.ts]: done

**Pull request merged**